lib/ui/mail_view_indv.dart

```diff
@@ -322,6 +331,23 @@ class MailPieceViewWidgetState extends State<MailPieceViewWidget> {
     }
   } //end _getLinks
 
+  //Function to strip out all line feeds \n to make sure test would wrap, then add it back
+  //for shorter blocks such as address or title blocks - currently set to 50 characters.
+
+  Future<String> _reformatMailPieceString(String x) async {
+    final find = '\n';
+    final replaceWith = ' ';
+    final String original = x;
+    final originalSplit = x.split('\n');
+    for(int i=0; i< originalSplit.length; i++) {
+        if(originalSplit[i].length < 50)
+          originalSplit[i] += '\n';
+        //originalSplit[i] = originalSplit[i].substring(0, originalSplit[i].length - 2);
```

Comment:
If we might not need this later, can you remove these commented out lines?

Reply:
Removed commented code